api/src/dtos/applications/application-lottery-total.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Expose } from 'class-transformer';
+import { IsDefined, IsNumber, IsString, IsUUID } from 'class-validator';
+import { ValidationsGroupsEnum } from '../../enums/shared/validation-groups-enum';
+
+export class ApplicationLotteryTotal {
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsUUID(4, { groups: [ValidationsGroupsEnum.default] })
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
+  listingId: string;
+
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  @IsUUID(4, { groups: [ValidationsGroupsEnum.default] })
+  @IsDefined({ groups: [ValidationsGroupsEnum.default] })
+  @ApiProperty()
+  multiselectQuestionId: string;
```

Comment:
since this is optional in the prisma schema, should this be optional here?

Reply:
Looking into now

Reply:
I think so, I was duplicating the behavior with `ApplicationLotteryPosition` - should I change `multiselectQuestionId` to be optional there as well?

Reply:
ooh yeah can we change that to be optional as well please!

Reply:
Can do!